app/src/main/res/drawable/ic_share.xml

```diff
@@ -1,12 +1,9 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="512dp"
-    android:height="512dp"
-    android:viewportWidth="512"
-    android:viewportHeight="512">
+    android:width="511dp"
```

Comment:
You can keep 512dp here probably, or even change it to the final size of the icon (it may apply to the other icons as well)

Reply:
Fixed. Must have been accidental change during svg import.